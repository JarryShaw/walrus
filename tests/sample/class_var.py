x = 66
y = 77


class A:
    global z
    print(z := 888)
    x = 99
    print(x := x + 1)
    print(y := x + 10)
    print(y + 100)
    y = 999
    print(y)


print(x)
print(y)
print(z)
print(A.__dict__.get('z'))
