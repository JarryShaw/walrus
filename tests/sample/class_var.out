100
110
210
66
77
